opentech/static_src/src/sass/apply/fancybox.scss

```diff
@@ -424,6 +428,13 @@ body.fancybox-iosfix {
 	transition: background-color .25s;
 	box-sizing: border-box;
 	z-index: 2;
+
+	.modal--secondary & {
+		color: white;
+		font-size: 40px;
+		margin-top: 12px;
+    	margin-right: 8px;
```

Comment:
indentation

Reply:
weird, that's not showing up in my editor.